packages/paste-website/src/pages/patterns/navigation/index.mdx

```diff
@@ -0,0 +1,276 @@
+
+### What’s an account, workspace, or user?
+
+At Twilio, we define accounts, users, and workspaces as:
+
+- **Account**: In Twilio Console, a group of resources that are administered and billed together. Similar to workspaces in Segment. [Read more about accounts.](https://support.twilio.com/hc/en-us/articles/360011132374-Getting-Started-with-Twilio-Accounts-and-Subaccounts)
+- **Workspace**: In Segment, a group of sources that are administered and billed together. Similar to accounts in Twilio Console. [Read more about workspaces.](https://segment.com/docs/guides/#whats-a-workspace)
+- **User**: A person with a login. Users can have access to multiple accounts or workspaces.
+
+### Layout and composition
+
+To decide where to place new navigation elements, use these general principles:
+
+- The **Sidebar** is where customers navigate between product features using [Sidebar Navigation](/components/sidebar-navigation), or where they control Sidebar visibility.
+- The **left side of the Topbar** is where customers navigate to account or workspace controls.
+- The **right side of the Topbar** is where customers find product- or experience-wide features, like search, [Product Switcher](/components/product-switcher), and [User Dialog](/components/user-dialog).
+- **Navigating within a product feature** should be handled at the page-level, with components like [In-Page Navigation](/components/in-page-navigation) and [Breadcrumb](/components/breadcrumb).
+
+<ResponsiveImage src={NavigationWireframe} aria-hidden="true" alt="" />
+
+### Accessibility
+
+The Sidebar and Topbar navigation components have the following considerations:
+
+- Ensure the [Skip links](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html) are [correctly implemented](/components/sidebar#skip-links). This ensures users can easily and quickly access the main content of a web page by bypassing repeated blocks of content such as navigation links.
+- Ensure the `SidebarNavigation` component is labelled with an accessible name.
+
+### Topbar ingredients
+
+<ResponsiveImage src={NavigationTopbar} aria-hidden="true" alt="" />
+
+- [Topbar](/components/topbar)
+- [Account Switcher](/components/account-switcher)
+- [Anchor](/components/anchor)
+- [Badge](/components/badge)
+- [Combobox](/components/combobox)
+- [Icons](/components/icons), particularly `ChevronDownIcon`, `SearchIcon`, `SupportIcon`, `UserIcon`
+- [Menu](/components/menu)
+- [Product Switcher](/components/product-switcher)
+- [Status Menu](/components/status-menu)
+- [User Dialog](/components/user-dialog)
+
+### Sidebar ingredients
+
+<ResponsiveImage src={NavigationSidebar} aria-hidden="true" alt="" />
+
+- [Sidebar container](/components/sidebar)
+- [Sidebar Navigation](/components/sidebar-navigation)
+- [Icons](/components/icons), mainly product and feature logo icons
+
+## Examples
+
+### Product-agnostic
+
+<StoryPreview
```

Comment:
idk when this was added, but the "Open in storybook" links are niceeee

Reply:
I can't remember exactly why I added it this feature, but it was for sidebar and topbar reasons